docs/components/LinkComponent.tsx

```diff
@@ -14,7 +14,10 @@ export const LinkComponent = forwardRef<HTMLAnchorElement, LinkComponentProps>(
 		// Use an `a` tag when linking externally
 		// Regex finds links starting with: `http://` | `https://` | `//`
 		const hrefAsString = typeof href === 'string' ? href : href?.pathname;
-		if (hrefAsString && /^(https?:\/\/|\/\/)/i.test(hrefAsString)) {
+		if (
+			(hrefAsString && /^(https?:\/\/|\/\/)/i.test(hrefAsString)) ||
+			hrefAsString?.includes('#')
+		) {
```

Comment:
Should the brackets be moved to make it this instead?

```suggestion
		if (
			hrefAsString &&
			(/^(https?:\/\/|\/\/)/i.test(hrefAsString) || hrefAsString?.includes('#'))
		) {
```

Reply:
Yep, no sense checking its existence twice. Don't need the optional chaining now. I'll update locally and push.

Reply:
- [ ] Make the check once

Reply:
Ah yeah, we don't need the optional. Even better